charts/operator/chart.go

```diff
@@ -34,53 +33,59 @@ var (
 	//go:embed values.yaml
 	defaultValuesYAML []byte
 
-	chartMeta helmette.Chart
+	// Chart is the go version of the operator helm chart.
+	Chart = gotohelm.MustLoad(chartYAML, defaultValuesYAML, render)
 )
 
 func init() {
 	must(scheme.AddToScheme(Scheme))
 	must(certmanagerv1.AddToScheme(Scheme))
 	must(monitoringv1.AddToScheme(Scheme))
+}
 
-	// NB: We can't directly unmarshal into a helmette.Chart as adding json
-	// tags to it breaks gotohelm.
-	var chart map[string]any
-	must(yaml.Unmarshal(chartYAML, &chart))
-
-	chartMeta = helmette.Chart{
-		Name:       chart["name"].(string),
-		Version:    chart["version"].(string),
-		AppVersion: chart["appVersion"].(string),
+func must(err error) {
+	if err != nil {
+		panic(err)
 	}
 }
 
-// ChartMeta returns a parsed version of redpanda's Chart.yaml.
-func ChartMeta() helmette.Chart {
-	return chartMeta
-}
+// render is the entrypoint to both the go and helm versions of the redpanda
```

Comment:
I'd prefer to not merge changes to the charts until we have CI running generation and unit tests for them. It's too easy for something to accidentally slip in and not get caught until MUCH later.